examples/utils.py

```diff
@@ -80,11 +79,20 @@ def get_distributed_model(model_path):
         kill_tgis_container_if_exists()
 
     # TODO: Enforce validation that TGIS is mounting the same model type
-    _CONFIGURED_BACKENDS.clear()
-    caikit.configure(
-        config_dict={"module_backends": {"priority": [TGISBackend.backend_type]}}
+
+    caikit.config.configure(
```

Comment:
I thought just `caikit.configure(...` should still also work but no big deal

Reply:
True. I didn't realize that, but good to know. Thanks